### PR TITLE
Enabling auto bump mechanism for contributions PRs

### PR DESCRIPTION
**Pull request by maimorag**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-6689)

## Description
The auto bump mechanism was added to content PRs in order to solve release notes and pack version conflicts.
Currently, it runs only on internal PRs, I was wondering why we don't use it for contributors' PRs either. As I understood from [Darya Koval](https://jira-hq.paloaltonetworks.local/secure/ViewProfile.jspa?name=dkoval), that was part of the initial task design - what was the rationale? [Tal Katzir](https://jira-hq.paloaltonetworks.local/secure/ViewProfile.jspa?name=tkatzir), [Dor Schwartz](https://jira-hq.paloaltonetworks.local/secure/ViewProfile.jspa?name=dschwartz).

I think it could benefit the contributors and reviewers during the review process.
We need to consider adding it to both External and Internal contributions PRs.

### CODE REVIEW

**Comment by maimorag** [comment]
[PR for testing](https://github.com/demisto/content/pull/28306)

**Comment by maimorag** [comment]
An update:
This PRs adds support of auto-bump to inside branches (all support types), but to outsides branches.
@dorschw @daryakoval

File: Utils/github_workflow_scripts/autobump_release_notes/skip_conditions.py [hunk 1]
```diff
@@ -411,7 +411,7 @@ def _has_conflict_on_given_files(
 
 
 class PackSupportCondition(MetadataCondition):
-    ALLOWED_SUPPORT_TYPE = Metadata.XSOAR_SUPPORT
+    ALLOWED_SUPPORT_TYPE = (Metadata.XSOAR_SUPPORT, Metadata.PARTNER_SUPPORT, Metadata.COMMUNITY_SUPPORT)
```
Comment (dorschw) [inline Utils/github_workflow_scripts/autobump_release_notes/skip_conditions.py:414]:
```suggestion
    ALLOWED_SUPPORT_TYPES = (Metadata.XSOAR_SUPPORT, Metadata.PARTNER_SUPPORT, Metadata.COMMUNITY_SUPPORT)
```

File: Utils/github_workflow_scripts/autobump_release_notes/skip_conditions.py [hunk 2]
```diff
@@ -434,7 +434,7 @@ def _check(
             or we should skip this pr from auto-bumping its release notes, with the reason why to skip.
         """
         support_type = self.branch_metadata.get(Metadata.SUPPORT)
-        if support_type != self.ALLOWED_SUPPORT_TYPE:
+        if support_type not in self.ALLOWED_SUPPORT_TYPE:
```
Comment (dorschw) [inline Utils/github_workflow_scripts/autobump_release_notes/skip_conditions.py:437]:
```suggestion
        if support_type not in self.ALLOWED_SUPPORT_TYPES:
```

**Pull request merged**